lxl-web/tailwind.config.js

```diff
@@ -192,6 +192,7 @@ export default {
 					'@apply p-4 sm:px-6': {}
 				}
 			});
+			addVariant('mobile-only', "@media screen and (max-width: theme('screens.sm'))");
```

Comment:
Do we need an extra utility class for mobile-only when we already have the possiblity to have classes as mobile first together with `sm:classes-here-for-larger-breakpoints` for larger breakpoints?

Reply:
`max-sm:classes-here` is another already existing alternative https://tailwindcss.com/docs/responsive-design#targeting-a-single-breakpoint

Reply:
Yes, `mobile-only` was ugly. Wanted to modify the button classes without having to then re-apply the original styling for the sm-screens and up.

But going for bigger buttons now it's not needed anyway.